packages/babel-jest/src/index.ts

```diff
@@ -119,10 +125,21 @@ const createTransformer = (options: any): Transformer => {
 
       const transformResult = babelTransform(src, babelOptions);
 
-      return transformResult || src;
+      if (transformResult && typeof transformResult.code === 'string') {
+        // @ts-ignore: why doesn't TS understand this?
```

Comment:
`transformResult` type inferred as `BabelFileResult` at declaration, will never get widened afterwards. Could work around it https://github.com/jeysal/jest/commit/290fcb6e8c75b202592493944a10f1ee2a72b925

Reply:
Thanks! I would have thought the `typeof` above was enough, but this works for me 🙂